evasquez26/angular4-startup
# S2-ONE Signin view design

## Story / Task

pending design for [ONE-30](https://bixlabs.atlassian.net/browse/ONE-30)


## What this PR does?

Adds design to login view


### Before 

No design for login view

### After

Login view has design applied

### How to test / reproduce

- While logged out, click sign in button
- Signin modal should appear with styles applied

### TODOs

### Related PRs
N/A

## Migrations

NO


Thank you, @evasquez26
